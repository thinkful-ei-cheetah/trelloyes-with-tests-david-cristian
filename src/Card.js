import React from 'react';
import './styles/Card.css';

export default function Card(props) {
  return (
    <div className='Card'>
      <h3>{props.title}</h3>
      <p>{props.content}</p>

      <button type='button'>
        Delete
      </button>
    </div>
  )
}