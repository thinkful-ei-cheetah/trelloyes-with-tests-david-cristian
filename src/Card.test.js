import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';


it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Card title="card-test" content="card-content" />, div)

    ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
        .create(<Card title="card-test" content="card-content" />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});