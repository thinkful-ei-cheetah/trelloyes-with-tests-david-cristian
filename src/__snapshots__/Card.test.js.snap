// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    card-test
  </h3>
  <p>
    card-content
  </p>
  <button
    type="button"
  >
    Delete
  </button>
</div>
`;
